src/webgpu/api/operation/command_buffer/copyTextureToTexture.spec.ts

```diff
@@ -86,7 +317,7 @@ g.test('t2t_non_compressed_color_formats')
         poptions('copyBoxOffsets', [
```

Comment:
I don't think there are any empty-copy cases in here either?

Reply:
I've put all the zero-copy cases into one test group "zero_copies", PTAL.